packages/webapp-alternative/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "webapp-alternative",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.3.0",
```

Comment:
Let's fix all versions (remove ^ sign). Because sometimes maintainers of packages break the package and you'll have a broken app without changes. So, if you fix dependencies it will be a safer approach.
Link to the description of versions

https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json

Reply:
Done